CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ the code was deployed.
 - Alert Type to the Dashboard.
 - Tracking of when sessions are first responded to (CU-hjwfx2).
 - `GET /alert/historicAlerts` endpoint (CU-hjwfx2).
+- Added cloud function call to siren in case of alert (CU-18r17vf)
```

Comment:
Could you please add a period of the end of this to be consistent with the other lines?